src/alexapi/device_platforms/orangepi.py

```diff
@@ -3,7 +3,7 @@
 import os
 import threading
 
-from pyA20.gpio import gpio as GPIO
+from pyA20.gpio import gpio as GPIO   # pylint: disable=import-error
```

Comment:
What do you need this for? The pyA20 library should be installed on dev machines as well as on Travis CI.